packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -398,36 +407,42 @@ Future<XcodeBuildResult> buildXcodeProject({
       ),
     );
   } else {
-    // If the app contains a watch companion target, the sdk argument of xcodebuild has to be omitted.
```

Comment:
This hunk is mostly whitespace changes.  The copy logic and watch companion adjustments were moved into the `buildAction == XcodeBuildAction.build` condition.